### PR TITLE
mariadb 10.3.7, mariadb@10.2 10.2.15 (new formula)

### PR DESCRIPTION
**Pull request by makigumo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
* update to MariaDB 10.3.7 (the first stable release of the 10.3 series).
* create new formula mariadb@10.2
* remove mariadb@10.2 alias
* create mariadb@10.3 alias

### CODE REVIEW

File: Formula/mariadb@10.2.rb [hunk 1]
```diff
@@ -0,0 +1,177 @@
+class MariadbAT102 < Formula
+  desc "Drop-in replacement for MySQL"
+  homepage "https://mariadb.org/"
+  url "https://downloads.mariadb.org/f/mariadb-10.2.15/source/mariadb-10.2.15.tar.gz"
+  sha256 "33de205158fc22fd8eb4e5770cc5ffa1cb4029f9c398dfd8c554ccb3e636ba11"
+
+  bottle do
```
Comment (commitay) [inline Formula/mariadb@10.2.rb]: Remove the bottle block.

File: Formula/mariadb@10.2.rb [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    server starting up correctly.
+
+    MySQL is configured to only allow connections from localhost by default
+
+    To connect:
+        mysql -uroot
+    EOS
+  end
+
+  plist_options :manual => "mysql.server start"
```
Comment (commitay) [inline Formula/mariadb@10.2.rb]: `plist_options :manual => "#{HOMEBREW_PREFIX}/opt/mariadb@10.2/bin/mysql.server start"`

**Review by commitay — CHANGES_REQUESTED** [review]

File: Formula/mariadb@10.2.rb (hunk 1 above)
Comment (makigumo) [reply Formula/mariadb@10.2.rb]: Done.

File: Formula/mariadb@10.2.rb (hunk 2 above)
Comment (makigumo) [reply Formula/mariadb@10.2.rb]: Done.

**Review by commitay — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thanks @makigumo !

**Pull request closed without merging**